--- orig/fully_connected.h	2021-06-28 11:25:04.090446372 +0200
+++ patched/fully_connected.h	2021-06-29 09:07:51.091522260 +0200
@@ -12,7 +12,16 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#ifndef TENSORFLOW_LITE_MICRO_KERNELS_FULLY_CONNECTED_H_
+/* Copyright 2021 Renesas Electronics
+
+Renesas modified this file partly.
+The modifications are framed within the following comments                    */
+
+    /* Modified by Renesas Electronics {  */
+
+    /* Modified by Renesas Electronics }  */
+
+/*============================================================================*/#ifndef TENSORFLOW_LITE_MICRO_KERNELS_FULLY_CONNECTED_H_
 #define TENSORFLOW_LITE_MICRO_KERNELS_FULLY_CONNECTED_H_
 
 #include <cstdint>
@@ -21,6 +30,10 @@
 #include "tensorflow/lite/c/common.h"
 #include "tensorflow/lite/kernels/internal/types.h"
 
+/* Modified by Renesas Electronics {  */
+#define CMSIS_NN
+/* Modified by Renesas Electronics }  */
+ 
 namespace tflite {
 
 struct OpDataFullyConnected {
